students/msemerni/hw-8/js/task-2.js

```diff
@@ -0,0 +1,25 @@
+//// Task 2
+//// Create an array with objects
+
+const createArraysWithObjects = (name1, name2) => {
+  let id = 1;
+  const arrayFromObject = [];
+  arrayFromObject.push({name: name1, id: id++});
+  arrayFromObject.push({name: name2, id: id++});
+  return arrayFromObject;
+}
+
+function createArraysWithObjectsUnlim  (names)  {
+  let id = 1;
+  const arrayFromObject = [];
+  [...arguments].map(name => {
+    arrayFromObject.push({name: name, id: id++});
```

Comment:
To achieve positive ids use prefix incrementing - [info](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Increment)
```suggestion
    arrayFromObject.push({name: name, id: ++id});
```